generator/src/test/java/com/scottlogic/deg/generator/helpers/StringGeneratorHelper.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Scott Logic Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.scottlogic.deg.generator.helpers;
+
+import com.scottlogic.deg.generator.generation.string.StringGenerator;
+import org.junit.Assert;
+
+import java.util.Iterator;
+
+import static org.hamcrest.Matchers.*;
+
+public enum StringGeneratorHelper {
+    ;
```

Comment:
semi colon can be removed

Reply:
as its an enum its required i think otherwise won't compile.  I tend to use uninstantiatable enums for Helpers with only static methods.  Some would argue that we should never have static methods but I'd argue that there are circumstances, like this, where its ok.

Reply:
it doesn't seem right to me that this would be an enum.

it's not used as an enum, so it's potentially confusing to see it referred to as such.